tinydrive/tinydrive-api/standalone.md

```diff
@@ -83,6 +142,28 @@ tinydrive.start({
 });
 ```
 
+## `tinydrive.controls.start`
+
+Start is similar to pick and browse, the main difference is that method doesn't have a way of closing the dialog by the user. This can be useful when you want to launch {{site.cloudfilemanager}} from an url where there is no application to insert the files into. This might be useful when you want to launch {{site.cloudfilemanager}} from an url but not part of a bigger application. For a complete list of available settings to pass in to this method check the [Picker settings](#pickersettings) section in this page.
```

Comment:
```suggestion
Start is similar to pick and browse, the main difference is that users do not have a way of closing the dialog with this method. This can be useful when you want to launch {{site.cloudfilemanager}} from an url where there is no application to insert the files into. This might be useful when you want to launch {{site.cloudfilemanager}} from an url but not as part of a bigger application. For a complete list of available settings to pass in to this method see [Picker settings](#pickersettings).
```